prov/efa/configure.m4

```diff
@@ -84,7 +84,16 @@ AC_DEFUN([FI_EFA_CONFIGURE],[
 	      ])
 	CPPFLAGS=$save_CPPFLAGS
 
-	AS_IF([test $efa_happy -eq 1 ], [$1], [$2])
+	dnl Check for ibv_is_fork_initialized() in libibverbs
+	AS_IF([test $efa_happy -eq 1],
+		[AC_CHECK_DECL([ibv_is_fork_initialized],
+			[AC_DEFINE([HAVE_IBV_IS_FORK_INITIALIZED],
+				[1], [Define to 1 if libibverbs has ibv_is_fork_initialized])],
```

Comment:
Please always define values to 0 or 1 (vs. just defining them or not defining them).  It's safer/better.

Reply:
I had gone with this definition to stay in line with the rest of the EFA provider, that seemed safer to me than mixing style.

Reply:
You guys are in charge of your own provider, but I'd (strongly) advocate for always defining to 0 or 1.  It's a defensive coding thing, and is used elsewhere in libfabric (I don't know if it's uniformly applied across libfabric... but my $0.02 is that it should be).

Reply:
I agree @jsquyres, I think mixing style is ok for this PR. We should fix other similar places too.